src/CLR/Core/TypeSystem.cpp

```diff
@@ -1865,6 +1865,12 @@ void CLR_RT_Assembly::DestroyInstance()
     }
 #endif
 
+    // check if header has to be freed (in case the assembly lives in RAM)
+    if (this->m_flags & CLR_RT_Assembly::FreeOnDestroy != 0)
```

Comment:
per @dougboling suggestion

```suggestion
    if ((this->m_flags & CLR_RT_Assembly::FreeOnDestroy) != 0)
```